### ISSUE TITLE
Some HTML entities are no longer replaced with v2.1.2

### ISSUE THREAD

**Issue by cbautista1002** [mozilla/bleach#379]
Hi! I'm currently using v2.0.0 and I have a unit test in my app that confirms that bleach works with my app as we expect. This tests confirms that given `'&eacute` a `é` is returned. Here's a code snippet:
```python
>>> import bleach
>>> bleach.__version__
'2.0.0'

>>> bleach.clean('&eacute')
'é'
```

This is no longer that case with v2.1.3 which I'd like to upgrade to.
```python
>>> import bleach
>>> bleach.__version__
'2.1.3'

>>> bleach.clean('&eacute')
'&amp;eacute'
```

I checked the [release notes](https://bleach.readthedocs.io/en/latest/changes.html) but I don't see something that would explain this. Unless if #143 is related.

I'm on python 3.6

Was this an intentional change? If so, then is there some new doc that can explain what I can expect here? Thanks!

**Comment by willkg** [comment]
I'm pretty sure that changed when we fixed issue #143. I think what's going on is that that's an ambiguous character entity and we decided to treat those like text when they're in a character context. That was like 9 months ago, though, so I don't remember the specifics.

This works fine with character references--just not ambiguous ones:

```
$ python
Python 3.6.5 (default, Apr  1 2018, 05:46:30) 
[GCC 7.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import bleach
>>> bleach.clean('&eacute;')
'&eacute;'
>>> bleach.clean('&eacute')
'&amp;eacute'
>>> 
```

Is it important for your app to use ambiguous character references? Can you add `;` to the end?

**Comment by cbautista1002** [comment]
Ok. Not a problem. It's not important for my app. I can switch. Just wanted to confirm that this was intentional and why. Thanks!
